spec/system/school/coaches_index_spec.rb

```diff
@@ -56,32 +56,60 @@
     expect(coach.public).to eq(false)
 
     # Edit the coach to add remaining fields.
-    find('p', text: new_coach_name).click
-
-    fill_in 'Connect Link', with: 'https://www.connect.com/xyz'
-    expect(page).to have_text("Since the coach profile isn't public, this won't be shown anywhere")
-    within('div[aria-label="public-profile-selector"]') { click_button 'Yes' } # Should the coach profile be public?
-    expect(page).not_to have_text("Since the coach profile isn't public, this won't be shown anywhere")
-    attach_file 'faculty[image]', File.absolute_path(Rails.root.join('spec/support/uploads/faculty/human.png')), visible: false
-    fill_in 'Name', with: updated_coach_name
-    expect(page).not_to have_field('Email')
-    fill_in 'Title', with: updated_coach_title
-    fill_in 'Affiliation', with: ''
-    click_button 'Update Coach'
-
-    expect(page).to have_text('Coach updated successfully')
-
-    expect(coach.reload.connect_link).to eq('https://www.connect.com/xyz')
+    find("p", text: new_coach_name).click
+
+    fill_in "Connect Link", with: "https://www.connect.com/xyz"
+    expect(page).to have_text(
+      "Since the coach profile isn't public, this won't be shown anywhere"
+    )
+    within('div[aria-label="public-profile-selector"]') { click_button "Yes" } # Should the coach profile be public?
+    expect(page).not_to have_text(
+      "Since the coach profile isn't public, this won't be shown anywhere"
+    )
+    attach_file "faculty[image]",
+                File.absolute_path(
+                  Rails.root.join("spec/support/uploads/faculty/human.png")
+                ),
+                visible: false
+    fill_in "Name", with: updated_coach_name
+    expect(page).not_to have_field("Email")
+    fill_in "Title", with: updated_coach_title
+    fill_in "Affiliation", with: ""
+    click_button "Update Coach"
+
+    expect(page).to have_text("Coach updated successfully")
+
+    expect(coach.reload.connect_link).to eq("https://www.connect.com/xyz")
     expect(coach.public).to eq(true)
     expect(user.avatar.attached?).to eq(true)
-    expect(user.avatar.filename).to eq('human.png')
+    expect(user.avatar.filename).to eq("human.png")
     expect(user.reload.name).to eq(updated_coach_name)
     expect(user.title).to eq(updated_coach_title)
     expect(user.affiliation).to eq(nil)
   end
 
-  scenario 'user who is not logged in gets redirected to sign in page' do
+  scenario "user who is not logged in gets redirected to sign in page" do
     visit school_coaches_path
-    expect(page).to have_text('Please sign in to continue.')
+    expect(page).to have_text("Please sign in to continue.")
+  end
+
+  scenario "school admin archives a coach" do
```

Comment:
Missing spec to check if `?stauts=archived` works.

Reply:
I think, I am checking on line number `111`
```ruby
find("span", text: "Exited Coaches").click
```